pennylane/qnodes/passthru.py

```diff
@@ -69,14 +69,14 @@ class PassthruQNode(BaseQNode):
             A Python function containing :class:`~.operation.Operation` constructor calls,
             and returning a tuple of measured :class:`~.operation.Observable` instances.
         device (~pennylane._device.Device): computational device to execute the function on
-        properties (dict[str, Any] or None): additional keyword properties for adjusting the QNode behavior
+        kwargs (dict[str, Any] or None): additional keyword arguments for adjusting the QNode behavior
```

Comment:
I would do the same thing you had in the decorator; remove this line, and have in it's place
```python
Keyword Args:
    use_native_type (bool): I don't know what this keyword argument does, but we should document it
```

Reply:
(_It's good that I double-checked the content here, almost went for just including it haha :smile:_)

**_tl;dr:_**
The only device using this right now is `default_tensor_tf` that uses this attribute for returning results without calling `Tensor.numpy()`.

The place where this is documented is in the `Device`  class as `default.tensor.tf`subclasses this along the line of inheritance.

I've used docstring from there to add it for:
* the `BaseQNode` with a default of `False`
* the `PassthruQNode` with a default of `True`

**_More details:_**

So this one seems to be a tough one, it seems to be used when passing the `return_native_type` argument in `base.py` to `device.execute`.

Ran two commands locally:
1. First looking for the occurences of `use_native_type`:
`grep -rnw . -e 'use_native_type' --include \*.py`
```python
./pennylane/pennylane/qnodes/base.py:775:        temp = self.kwargs.get("use_native_type", False)
./pennylane/pennylane/qnodes/passthru.py:77:        kwargs.setdefault("use_native_type", True)
```
2. Then looking for `return_native_type`:
`grep -rnw . -e 'return_native_type' --include \*.py`
```python
./pennylane/pennylane/_device.py:165:            return_native_type (bool): If True, return the result in whatever type the device uses
./pennylane/pennylane/qnodes/base.py:778:            ret = self.device.execute(self.circuit, return_native_type=temp)
./pennylane/pennylane/qnodes/base.py:784:                return_native_type=temp,
./pennylane/pennylane/beta/plugins/default_tensor_tf.py:356:        if kwargs.get("return_native_type", False):
```

These are results with having most plugins locally as well from the directory where the command was run.

Reply:
Thanks Antal! That is great, and helps reduce documentation debt